Storage/SQL/JoinedHistoryVisitsTable.swift

```diff
@@ -37,7 +37,7 @@ class JoinedHistoryVisitsTable: Table {
         if (cursor.count != 1) {
             return nil
         }
-        return (cursor[0] as Site).id
+        return (cursor[0] as? Site)?.id
```

Comment:
You have one choice here: crash if it's not a `Site` (`as!`), or chain through the `nil` (`as?`). Again, given that this function already returned `nil` in some cases, I opted to just ignore any malformed data.